src/python_testing/TC_RVCOPSTATE_2_1.py

```diff
@@ -190,11 +190,11 @@ async def test_TC_RVCOPSTATE_2_1(self):
                 self.print_step("7e", "Manually put the device in the unable to complete operation error state")
                 input("Press Enter when done.\n")
                 await self.read_and_validate_operror(step="7f", expected_error=Clusters.OperationalState.Enums.ErrorStateEnum.kUnableToCompleteOperation)
-            if self.check_pics("RVCOPSTATE.S.M.ERR_COMMAND_INVALID_STATE"):
+            if self.check_pics("RVCOPSTATE.S.M.ERR_COMMAND_INVALID_IN_STATE"):
```

Comment:
`src/python_testing/TC_OPSTATE_2_1.py` has this problem too.  Want to fix that while you're here, or in a followup?

Reply:
I'll fix that in this PR, will do it tonight.